src/dummywallet.cpp

```diff
@@ -28,6 +28,7 @@ void DummyWalletInit::AddWalletOptions(ArgsManager& argsman) const
         "-addresstype",
         "-avoidpartialspends",
         "-changetype",
+        "-discardfee=<amt>",
```

Comment:
AssertionError: Please add {'-consolidatefeerate'} to the hidden args in DummyWalletInit::AddWalletOptions

Reply:
Oops, fixed.